build/scripts/test-build-correctness.ps1

```diff
@@ -25,15 +25,15 @@ try {
     # Need to parse out the current NuGet package version of Structured Logger
     $structuredLoggerPath = Join-Path (Get-PackageDir "Microsoft.Build.Logging.StructuredLogger") "lib\net46\StructuredLogger.dll"
     $configDir = Join-Path $binariesDir $config
-    $logPath = Join-Path $configDir "build.xml"
+    $logPath = Join-Path $configDir "roslyn.buildlog"
     $solution = Join-Path $repoDir "Roslyn.sln"
 
     if ($msbuild -eq "") {
         $msbuild = Ensure-MSBuild
     }
 
     Write-Host "Building Roslyn.sln with logging support"
-    Exec-Command $msbuild "/v:m /m /p:Configuration=$config /logger:StructuredLogger,$structuredLoggerPath;$logPath /nodeReuse:false /p:DeployExtension=false $solution"
+    Exec-Command $msbuild "/noconlog /v:m /m /p:Configuration=$config /logger:StructuredLogger,$structuredLoggerPath;$logPath /nodeReuse:false /p:DeployExtension=false $solution"
```

Comment:
Removing the console output should speed the build up a bit, and we have the full log anyway if we need it. I can bring it back if you like?